### PR TITLE
Add release notes for 2.1.12

### PR DESCRIPTION
**Pull request by jugarrit**
@lemieux 
@Mario54 
@alavers 
@jpjoyal

### CODE REVIEW

**Comment by lemieux** [comment]
:+1:

File: release_notes/v2.1.12.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Additions
+
+Device info 'Looking at' will now be updated automatically on hashchange or history state modification
```
Comment (alavers) [inline release_notes/v2.1.12.md]: nit, clarify that Looking at means browser URL

File: release_notes/v2.1.12.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+# Bug fixes
+
+Now automatically re-fetching the conversation on web socket reconnect to prevent message loss
+Fixed a bug where user properties were overridden when conversation was fetched
```
Comment (Mario54) [inline release_notes/v2.1.12.md]: It would be more accurate to say
> [...] were overridden when calling `Smooch.getConversation()` succeeded.

**Comment by Mario54** [comment]
:+1:

**Comment by alavers** [comment]
:+1:

**Pull request merged**